business_requirement_deliverable_cost/__openerp__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Business Requirement Deliverable Cost Control",
     "summary": "Control the cost of your Business Requirements",
-    "version": "8.0.1.0.1",
+    "version": "8.0.1.0.2",
```

Comment:
I think we should bump to 8.0.2.0.0 because this modules implies a manual migration of the data